wDeveloper/OpenSourceProjects.md

```diff
@@ -20,6 +20,11 @@ Interject hosts a variety of open-source projects aimed at providing useful tool
 - [Script My SQL Server](#scriptmysqlserver) : Set of scripts to automate MySQL server interactions
 - [Powershell DB Tools GUI](#powershell-db-tools-gui) : Simplify scripting out DB objects and executing SQL scripts
 
+<blockquote class=highlight_note>
+<b>Note:</b> These projects are not currently maintained. Use at your own discretion. 
```

Comment:
we should probably change this to the following since we do make patches and occasional fixes to some of them but not regularly.

```
Some of these projects are not actively maintained.
```